### PR TITLE
add transparent hugepage enabled information

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Related PR: https://github.com/pingcap/sysutil/pull/23

example in TiDB:

```sql
information_schema> select * from `CLUSTER_SYSTEMINFO` where name = 'transparent_hugepage_enabled';
+------+------------------+-------------+-------------+------------------------------+------------------------+
| TYPE | INSTANCE         | SYSTEM_TYPE | SYSTEM_NAME | NAME                         | VALUE                  |
+------+------------------+-------------+-------------+------------------------------+------------------------+
| tikv | 172.16.5.40:4019 | system      | kernel      | transparent_hugepage_enabled | always madvise [never] |
+------+------------------+-------------+-------------+------------------------------+------------------------+

```

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

- No.

### Release note 

- No.

### CODE REVIEW

**Comment by crazycs520** [comment]
@breeswish @lonng PTAL

File: src/server/service/diagnostics/sys.rs [hunk 1]
```diff
@@ -479,6 +480,21 @@ fn get_sysctl_list() -> HashMap<String, String> {
         .collect()
 }
 
+fn get_transparent_hugepage() -> ServerInfoItem {
```
Comment (lonng) [inline src/server/service/diagnostics/sys.rs:483]: This function should return `Option<ServerInfoItem>`.

Comment (breeswish) [reply src/server/service/diagnostics/sys.rs:483]: Agree

**Review by breeswish — APPROVED** [review]
Rest LGTM

File: src/server/service/diagnostics/sys.rs (hunk 1 above)
Comment (crazycs520) [reply src/server/service/diagnostics/sys.rs]: addressed, PTAL

**Review by breeswish — APPROVED** [review]

**Review by breeswish — APPROVED** [review]

**Pull request merged**